resources/seqr_metadata_templates/README.md

```diff
@@ -0,0 +1,264 @@
+
+2. [csv template](csv_templates/individuals_metadata_template.csv)<br />
+
+4.2.&emsp;All information relating to individuals should be documented in a single *individuals_metadata_template* file.
+
+**Note:** You should only have **one** *individuals_metadata_template* file. <br />&emsp;&emsp;&ensp;&ensp;This single file can contain as many individuals as described in your cohort/dataset. <br />&emsp;&emsp;&ensp;&ensp;Do not create separate *individuals_metadata_template* files for each family in your cohort/dataset.
+
+4.3.&emsp;Populate the *individuals_metadata_template* according to Table 6.<br />
+&emsp;&emsp;&ensp;&nbsp;An example is given below in Table 7.
+
+**Note**: Only populate the fields that you have information for. Not every field needs to be populated in this template file.  <br />&emsp;&emsp;&ensp;&ensp;The more information you provide in the file, the better your experience will be in seqr.
+
+4.4.&emsp;Ensure that the *individuals_metadata_template* file is shared alongside your transfer.
+
+<br />
+
+#### **Table 6:** Data dictionary for *individuals_metadata_template* file describing inputs for the template fields. <!-- omit from toc -->
+
+| **Field label** | **Allowed Values** | **Notes** |
+|:---:|:---:|:---|
+| Family ID | Alphanumeric family ID | The   combination of family ID and individual ID should uniquely identify an individual. |
+| Individual ID | Alphanumeric individual ID |  |
+| HPO Terms (present) | Comma-separated list of HPO codes for phenotypes present in this individual | This field should have the HPO codes, not the descriptions. |
+| HPO Terms (absent) | Comma-separated list of HPO codes for phenotypes not present in this individual | This field should have the HPO codes, not the descriptions. |
+| Birth Year | Numeric year of birth. E.g. 2010 | If you have collected a DOB, e.g. 01-01-2001, please only include the **year** component. |
+| Death Year | Numeric year of death, if applicable. <br /> Leave blank otherwise. | If you have collected a DOD, e.g. 01-01-2001, please only include the **year** component. |
+| Age of Onset | **One of the following:** <br /><br />Congenital onset, <br />Embryonal onset, <br />Fetal onset, <br />Neonatal onset, <br />Infantile onset, <br />Childhood onset, <br />Juvenile onset, <br />Young adult onset, <br />Adult onset, <br />Middle age onset, <br />Late onset | *This is a rough suggestion, with no clinical source.*<br /> <br /> **Congenital onset:** conception to birth <br /> **Embryonal onset:** conception to 8 wks gestation <br /> **Fetal onset:** 9 wks gestation to birth<br />  **Neonatal onset:** birth to 1 month, <br /> **Infantile onset:** birth to 1 year<br /> **Childhood onset:** < 5 years<br /> **Juvenile onset:** < 17 years<br /> **Young adult onset:** < 25 years<br /> **Adult onset:** < 36 years<br /> **Middle age onset:** < 55 years<br /> **Late onset:** > 55 years
+| Individual Notes | Alphanumeric characters |   |
```

Comment:
Should any of these empty description fields be filled in, or are they ok blank?

Reply:
> Should any of these empty description fields be filled in, or are they ok blank?

All fields are optional other than `Family ID` and `Individual ID`.

However, now that you mention it it would be good to move `Individual Notes` up to the third row and add a comment that this is a free text field that will be displayed in seqr. It can be a good place to describe the individual's presentation and any features important for variant curation.